### PR TITLE
feat: Don't post nests with poor spawn rates

### PR DESCRIPTION
**Pull request by clburlison**
Nests that average really poor spawn rates aren't valuable IMO. This allows users to adjust what will be included in the Discord webhook.

### CODE REVIEW

File: analyze_nests.py [hunk 1]
```diff
@@ -870,6 +873,8 @@ def discord_webhook():
         for b_area in sorted_basic_areas.values():
             if config['dc-ignore-unnamed'] and (b_area["name"] == config["default_park_name"]):
                 continue
+            if int(b_area["pokemon_avg"]) < int(config["dc-min-spawns-for-post"]):
```
Comment (M4d40) [inline analyze_nests.py]: why int and not float? so you can also check for 2.5

File: analyze_nests.py [hunk 2]
```diff
@@ -870,6 +873,8 @@ def discord_webhook():
...
+                break
```
Comment (M4d40) [inline analyze_nests.py]: please use `continue`, else it won't iterate through all areas

File: analyze_nests.py (hunk 1 above)
Comment (clburlison) [reply analyze_nests.py]: Done

**Comment by clburlison** [comment]
Updates pushed. My test area was too small to notice it exiting.

**Pull request merged**